changelog.d/gh-8408.added

```diff
@@ -0,0 +1,17 @@
+Added support in Ruby, Julia, and Rust to match implicit return statement inside functions.
+
+For example:
+
+```julia
+return 0
+```
+
+can now match 0 in
+
+```julia
+function f()
+  0
+end
+```
+
+This matching is enabled by default and can be disabled with the rule option `implicit_return`.
```

Comment:
You checked this box:
>  Documentation is up-to-date

but have you opened a PR to semgrep-docs or added a ticket to the Documentation project? (Just checking.)

Reply:
Oh, my bad! Thanks for checking.

I misunderstood "documentation". I thought it was about documenting the code, e.g. adding comments explaining things, etc, but now that I look at it again, it seems to fall into the "code is evident to future readers" bullet point.

I will add a ticket to track it + make changes to semgrep-docs.